packages/integration-react/report.api.md

```diff
@@ -55,6 +55,20 @@ export class ScmAuth implements ScmAuthApi {
     bitbucketAuthApi: OAuthApi,
     options?: {
       host?: string;
+      scopeMapping?: {
+        default?: string[];
+        repoWrite?: string[];
+      };
+    },
+  ): ScmAuth;
+  static forBitbucketServer(
```

Comment:
I'm a bit hesitant to adding a new static method specifically for Bitbucket Server, isn't that product deprecated and being replaced by BitBucket Datacenter? I think the new scopeMapping you added to the other method alongside docs for bitbucket server on which scopes to set is the way to go.